drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -423,26 +424,44 @@ function DragAndDropTemplates(configuration) {
             },
             [
                 h(
+                  (ctx.last_action_correct ? 'div.popup-header-icon' : 'div.popup-header-icon.popup-header-icon-incorrect'),
+                  {},
+                  [
+                    h(
+                      (ctx.last_action_correct ? 'span.icon.fa.fa-check-circle' : 'span.icon.fa.fa-exclamation-circle'),
+                      {
+                            attributes: {
+                            'aria-hidden': true
```

Comment:
Nit: I *think* some spaces would be nice here.

Reply:
@UmanShahzad I am not very familiar with how JS formatting should be done. What do you have in mind for this?

Reply:
Usually no matter where you are, you'd format things according to the surrounding code, because consistency tends to be preferred.